odh-operator-image/Dockerfile

```diff
@@ -0,0 +1,10 @@
+ARG version=v0.8.0
+FROM quay.io/opendatahub/opendatahub-operator:${version}
+ARG version
+
+USER root
+RUN microdnf install -y wget && microdnf clean all && rm -rf /var/cache/yum
+RUN mkdir /manifests && chgrp 0 /manifests && chmod g+rwX /manifests
```

Comment:
We should follow LFS, I'd put it under `/opt`

Reply:
Done